README.md

```diff
@@ -295,6 +295,11 @@ All you need to do is specifying a few environment variables (optionally point t
 Note that metadata filters in queries are not yet supported.
 For detailed setup instructions, refer to [`/docs/providers/llama/setup.md`](/docs/providers/llama/setup.md).
 
+#### Postgres
+
+[PostgresML](https://www.postgresml.org) is an open-source machine learning extension that supports large language modeling and traditional ML workloads on [Postgres](https://www.postgresql.org) database. You can sign up for a [free GPU-powered Postgres database](https://postgresml.org/signup) or [Docker](https://github.com/postgresml/postgresml#docker)
```

Comment:
I think maybe we should just mention PgVector for this use case, since that's all that's really necessary.